src/Fixer/PhpTag/NoShortEchoTagFixer.php

```diff
@@ -12,16 +12,17 @@
 
 namespace PhpCsFixer\Fixer\PhpTag;
 
-use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\AbstractProxyFixer;
+use PhpCsFixer\Fixer\DeprecatedFixerInterface;
 use PhpCsFixer\FixerDefinition\CodeSample;
 use PhpCsFixer\FixerDefinition\FixerDefinition;
-use PhpCsFixer\Tokenizer\Token;
-use PhpCsFixer\Tokenizer\Tokens;
 
 /**
  * @author Vincent Klaiber <hello@vinkla.com>
+ *
+ * @deprecated in 2.16, proxy to ShortEchoTagFixer
```

Comment:
Version is now out of date, I don't think we need it at all.

Reply:
Done: 8ec8c60451e2e31844b1c0f81c1627b759c716ed